easybuild/easyblocks/t/tensorflow.py

```diff
@@ -446,7 +453,9 @@ def configure_step(self):
         # and will hang forever building the TensorFlow package.
         # So limit to something high but still reasonable while allowing ECs to overwrite it
         if self.cfg['maxparallel'] is None:
-            self.cfg['parallel'] = min(self.cfg['parallel'], 64)
+            # Seemingly Bazel around 3.x got better, so double the max there
+            bazel_max = 64 if get_bazel_version() < '3.0.0' else 128
```

Comment:
I would prefer if we bump the Bazel version requirement a bit here, for example to `4.0`, unless you have a clear reference as to why this makes sense for Bazel 3.x too...
This is mainly to avoid introducing regressions, and having to re-test a wide range of TensorFlow easyconfigs with this updated PR (we use Bazel 3.x from `TensorFlow-2.3.1-foss-2019b.eb` onwards - that's 18 easyconfigs to test - with Bazel 4.x it's only `TensorFlow-2.8.4-foss-2021b.eb` currently)

Reply:
I was already cautious because this more or less arbitrary limitation was introduced when we used Bazel 0.x. Bazel 3.7 has quite a few changes that I'm using at other places so 3.x seemed like a good choice for a major version to make the switch.

I'm not even sure it really was Bazel causing the trouble. Furthermore it shouldn't affect too much and it has been a few years that passed.
I actually "needed" that for TF 2.7 when creating TF 2.8 to check for differences in behavior. TF 2.7 uses Bazel 3.7 hence the choice for 3 as the major version.

I'll kick off a few tests on one of the systems that were the cause for this limitation (192 core PPC), 96 core x86 seems to work fine. If those pass it should be ok, shouldn't it?